XTS/raw-xts-api-demo/test/pom.xml

```diff
@@ -62,6 +62,12 @@
             <type>pom</type>
             <scope>test</scope>
         </dependency>
+<!--         update htmlunit-core-js dependency version that is already included in graphene-webdriver but at version 2.13 -->
+        <dependency>
+            <groupId>net.sourceforge.htmlunit</groupId>
+            <artifactId>htmlunit-core-js</artifactId>
+            <version>2.64.0</version>
+        </dependency>
```

Comment:
Have you tried to update `graphene-webdriver`?

Reply:
Thanks @jmfinelli , I update the pom and I removed the graphene-webdriver that I found out it was not necessary. In fact, only the arquillian-drone-webdriver-depchain sub-dependency is now used.